be/src/vec/functions/uuid.cpp

```diff
@@ -0,0 +1,68 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "udf/udf.h"
+#include "vec/data_types/data_type_string.h"
+#include "vec/functions/simple_function_factory.h"
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_generators.hpp>
+#include <boost/uuid/uuid_io.hpp>
+
+namespace doris::vectorized {
+class Uuid : public IFunction {
+public:
+    static constexpr auto name = "uuid";
+
+    static FunctionPtr create() { return std::make_shared<Uuid>(); }
+
+    String get_name() const override { return name; }
+
+    bool use_default_implementation_for_constants() const override { return false; }
+
+    size_t get_number_of_arguments() const override { return 0; }
+
+    bool is_variadic() const override { return true; }
+
+    DataTypePtr get_return_type_impl(const DataTypes& arguments) const override {
+        return std::make_shared<DataTypeString>();
+    }
+
+    Status execute_impl(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
+                        size_t result, size_t input_rows_count) override {
+        auto col_res = ColumnString::create();
+        ColumnUInt8::MutablePtr col_null_map_to;
+        col_null_map_to = ColumnUInt8::create();
+        auto& vec_null_map_to = col_null_map_to->get_data();
+        vec_null_map_to.resize_fill(input_rows_count, 0);
+
+        boost::uuids::random_generator generator;
+        for (int i = 0; i < input_rows_count; i++) {
+            std::string uuid = boost::uuids::to_string(generator());
+            col_res->insert_data(uuid.c_str(), uuid.length());
+        }
+
+        block.get_by_position(result).column =
+                ColumnNullable::create(std::move(col_res), std::move(col_null_map_to));
```

Comment:
`ALWAYS_NOT_NULLABLE` means the result column is a pure ColumnString instead of ColumnNullable

Reply:
oh,oh,i fix it